### DISCUSSION TITLE
Does Supabase Realtime uses long polling fallback over WS?

### DISCUSSION THREAD

**Discussion by mrzslr** [supabase/supabase#17644] [Questions] [answered]
Hi,

Recently, I realized, that the Firebase Realtime Database sometimes switches to long polling instead of web socket due the poor connection and refreshing. Does Supabase have a similar approach or does it only use web socket whatsoever?

Thanks.

**Comment by ticmaisdev** [comment]
I need that answer, as well

**Reply by mrzslr** [reply, marked as answer]
We have used it and it doesn't fortunately. it's only using the WebSocket it seems.

**Reply by ticmaisdev** [reply]
so sad. supabase seems to bee amazing
